operations/research-and-development/product/release-process/playbooks-release.md

```diff
@@ -2,7 +2,7 @@
 
 This document outlines the Playbooks release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
 
-Every two weeks, the team runs through a playbook to cut a new release of the plugin, smoke test it, and add it to the marketplace. That gets pushed to cloud immediately \(via split.io\), and any self-managed customers who desire the upgrade may do so from the plugin marketplace.
+Every month, the team runs through a playbook to cut a new release of the plugin, smoke test it, and add it to the marketplace. That gets pushed to cloud immediately \(via split.io\), and any self-managed customers who desire the upgrade may do so from the plugin marketplace.
```

Comment:
Playbooks has recently moved to `mattermost-server`, built-in as product. I wonder if plugin release is still accurate or has to be updated. Same case about the [Boards release doc](https://github.com/mattermost/mattermost-handbook/blob/amyblais-patch-2/operations/research-and-development/product/release-process/boards-release.md). Maybe something for @sbishel to confirm?

Reply:
Monorepo didn't make it to v7.10 release, so I'll let Scott to confirm how this works now.